### ISSUE TITLE
RFE:  Add PVC mount for custom NAR directory

### ISSUE THREAD

**Issue by nnachefski** [guimou/nifi-on-openshift#11]
Add a PVC mount for the /opt/nifi/nifi-current/extensions directory so that custom NARs persist between restarts.

**Comment by guimou** [comment]
Good idea! I'll try to find some time to do it. Unless you beat me to it with a PR!

**Comment by nnachefski** [comment]
I have the code ready but am unable to create a branch in your repo.

```
ERROR: Permission to guimou/nifi-on-openshift.git denied to nnachefski.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

**Comment by guimou** [comment]
Oh, you should fork the repo, and submit the PR from your the fork. Like this you will be able to modify, branch,... as you like as it will be your own repo.
Sorry, I won't change direct accesses for now as this repo will soon move to opendatahub-contrib.

**Comment by nnachefski** [comment]
PR submitted.

https://github.com/guimou/nifi-on-openshift/pull/12/commits/709767565bd5c4f0c716cde2acc32a061348d6f7

**Comment by guimou** [comment]
Feature implemented and merged in #12

**Comment by nnachefski** [comment]
I think there is still some config that needs to be added here (volumes?).  It's hard for me to test via Kubeflow.  Also, i re-deployed nifi via the Open Data Hub operator KfDef but still dont see the "extensions" change.

**Comment by guimou** [comment]
?-1 The volumes were there in your PR
?-2 Deploy with the ODH operator? Although this is conceptually feasible as it's a KfDef after all, that's not how I'm using this. If you are redploying on an existing installation, I'm also not sure reapplying the KfDef will kick off the creation of the volumes, and the modification of the StatefulSet. You may have to modify it manually.

**Comment by guimou** [comment]
Fixed in #13.
I tested it, the extensions are persisted and loaded when the pod restarts.
